charts/bookkeeper/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 name: pravega
```

Comment:
Why is this pravega? This is a bookkeper chart yaml... please provide a suitable name.

Reply:
done